pom.xml

```diff
@@ -651,7 +650,7 @@
         <javax.activation.groupId>javax.activation</javax.activation.groupId>
         <javax.activation.artifactId>activation</javax.activation.artifactId>
         <javax.persistence.version>1.0.2</javax.persistence.version>
-        <jaxb.version>2.2.11</jaxb.version>
+        <jaxb.version>2.3.1</jaxb.version>
```

Comment:
is it used elsewhere?

Reply:
seems in `mailet/mailetcontainer-camel` and `protocols/jwt`

Reply:
Maybe it's not really used anymore by those modules... let me check

Reply:
swagger is using jaxb as well it seems, I prefer to not touch it